pom.xml

```diff
@@ -64,107 +64,102 @@
     <properties>
         <revision>2.17</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.138.4</jenkins.version>
+        <jenkins.version>2.176.4</jenkins.version>
         <java.level>8</java.level>
         <useBeta>true</useBeta>
-        <workflow-cps-plugin.version>2.71</workflow-cps-plugin.version>
-        <git-plugin.version>4.0.0-rc</git-plugin.version>
-        <scm-api-plugin.version>2.6.3</scm-api-plugin.version>
         <subversion-plugin.version>2.13.0</subversion-plugin.version>
-        <workflow-scm-step-plugin.version>2.7</workflow-scm-step-plugin.version>
         <workflow-multibranch-plugin.version>2.21</workflow-multibranch-plugin.version>
```

Comment:
needs adding to bom, will try take a look, but shouldn't block this

Reply:
Included in https://github.com/jenkinsci/bom/pull/179, FYI.